src/cmr/concepts/collection.js

```diff
@@ -124,6 +125,7 @@ export default class Collection extends Concept {
       'granule_data_format_h',
       'has_granules_or_cwic',
       'has_granules',
+      'has_opendap_url', 
```

Comment:
```suggestion
      'has_opendap_url',
```


just remove the whitespace after 128